Spanish Translation

FluffyHyena opened the pull request:
Added a Spanish translation both to the widgets and the menu, but can't test if it works as I don't have the means to compile the app. I hope it's ok!

tuur29 commented:
Thanks a lot! 🙂

On first glance it all looks fine, I'll test it as soon as possible and send you some screenshots to review, once everything looks good I'll merge this.

tuur29 commented:
I've run the app everything seems to work fine, but it seems like you (and I) forgot to add a few things:

- make sure all the [`pref_list_language_titles`](https://github.com/fluffyhyena/fuzzyclock/blob/dbbcf0321c7e1eecc7e0a9b2356b83a53d6318e4/shared/src/main/res/values/strings.xml#L56) lists (x3) have a "Spanish" value (not manually selectable otherwise), keep in mind that the order should be the same everywhere (same as `pref_list_language_values` in arrays.xml). You can use "Spaans" as a Dutch/NL translation.
- The label "[Visible](https://github.com/FluffyHyena/fuzzyclock/blob/master/shared/src/main/res/values-es/strings.xml#L96)" doesn't seem to be translated, or perhaps this is the same as English? (See screenshot).
- Could you add a comment in this PR with Spanish values for [pref_list_fontFamilyLimited_titles](https://github.com/fluffyhyena/fuzzyclock/blob/c635cc5f3652b0e50c5b179cb298adce0508c10d/shared/src/main/res/values/arrays.xml#L138) and [pref_list_emphasis_titles](https://github.com/fluffyhyena/fuzzyclock/blob/c635cc5f3652b0e50c5b179cb298adce0508c10d/shared/src/main/res/values/arrays.xml#L157)? It seems I've defined them in the wrong place and because of it they are currently not translatable. If you'd be so kind to translate them here in a comment, I'll add them at a later date when I fix this.

This is some test output for you to check:

```
00:00 -> docey cinco 
00:01 -> docey cinco 
01:03 -> unay cinco 
02:05 -> dosy cinco 
03:07 -> tresy diez 
04:10 -> cuatroy diez 
05:14 -> cincoy cuarto 
06:15 -> seisy cuarto 
07:16 -> sietey veinte 
08:20 -> ochoy veinte 
09:29 -> nuevey media 
09:30 -> nuevey media 
09:31 -> nuevemenos veinticinco 
09:35 -> diezmenos veinticinco 
09:36 -> diezmenos veinte 
10:41 -> oncemenos cuarto 
11:50 -> docemenos diez 
12:53 -> unamenos cinco 
12:55 -> unamenos cinco 
13:57 -> dos
14:59 -> tres
15:00 -> tresy cinco 
23:50 -> docemenos diez 
```

And some screenshots:

![Screenshot_20191202-221110](https://user-images.githubusercontent.com/2472626/69996795-0e9f2100-1553-11ea-978e-a8efef287c01.jpg)

![Screenshot_20191202-221115](https://user-images.githubusercontent.com/2472626/69996797-12cb3e80-1553-11ea-9118-1303985c9642.jpg)


![Screenshot_20191202-221123](https://user-images.githubusercontent.com/2472626/69996801-152d9880-1553-11ea-8fd1-7b7145153d21.jpg)


Time was 22:15:
![Screenshot_20191202-221512](https://user-images.githubusercontent.com/2472626/69996808-178ff280-1553-11ea-9159-769cc11f3eec.jpg)

Thanks again!

FluffyHyena commented:
Ok! In this order:

- Woops! Fixed! It should be alright now, thank you!

- Visible is the same in both languages! It was like "time to edit this! Oh wait, it doesn't need to be edited"

- Sure thing! Fortunately we use the same names for fonts than English, but the names for the emphasis are, in this order: Normal, Negrita, Cursiva, Negrita Cursiva. Hope that helps!

Also I just fixed an error in the display where I forgot to move the space string between the hour and minute string and I went and made the minutes more like the base English code rather than adding/removing anything sooo it should be ok now! Let me know of anything else!
